.github/workflows/ci.yml

```diff
@@ -36,7 +36,7 @@ jobs:
       - name: Generate code coverage
         env:
           RUST_BACKTRACE: "1"
-        run: cargo llvm-cov --all-features --lcov --output-path lcov.info
+        run: cargo llvm-cov --all-features --lcov --ignore-filename-regex tests --output-path lcov.info
```

Comment:
The --ignore-filename-regex is not needed and the `tests` directory should be excluded by default wherever it is located.
https://github.com/taiki-e/cargo-llvm-cov/blob/3f21ce8c7e21b38bf85112757cd119af8360af76/src/main.rs#L1227